info.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "sOCUtilities",
+    "version": "1.0.0",
+    "fsrMinCompatibility": "7.4.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "SOC Utilities",
+    "dependencies": [],
+    "prerequisite": "",
+    "publisher": "Fortinet CSE",
+    "description": "This Solution pack is designed around utilities to aid in the use of SOAR",
```

Comment:
Also we need to add below key;
"certified": "false",

Reply:
This has been fixed